config/routes.rb

```diff
@@ -41,6 +41,7 @@
   constraints(:id => %r{[^/]+}) do
     get 'cockpit/host_ssh_params/:id', to: 'cockpit#host_ssh_params'
   end
+  get 'cockpit/redirect', to: 'cockpit#redirect'
```

Comment:
@mvollmer updating the redirect endpoint: you might need to update it in the session script